catalog/spatial/registry/registry-report-viewer/src/main/java/org.codice.ddf.registry.report.viewer/RegistryReportViewer.java

```diff
@@ -0,0 +1,486 @@
+
+            organizationInfo.put(CONTACT_INFO, contactInfo);
+            organizationInfo.put(CUSTOM_SLOTS, getCustomSlots(organizationType.getSlot()));
+            organizationMap.put(orgName, organizationInfo);
+        }
+
+        return organizationMap;
+    }
+
+    private Map<String, Object> getContactInfo(RegistryPackageType registryPackage) {
+        Map<String, Object> contactMap = new HashMap<>();
+        List<PersonType> persons = registryPackageTypeHelper.getPersons(registryPackage);
+        for (PersonType person : persons) {
+            Map<String, Object> personMap = new HashMap<>();
+            Map<String, Object> contactInfo = new HashMap<>();
+
+            PersonNameType personName = person.getPersonName();
+            ArrayList<String> nameArray = new ArrayList<>();
+
+            if (personName == null) {
+                break;
+            }
+            addNonEmptyValue(nameArray, personName.getFirstName());
+            addNonEmptyValue(nameArray, personName.getMiddleName());
+            addNonEmptyValue(nameArray, personName.getLastName());
+
+            String name = StringUtils.join(nameArray, " ");
+
+            addNonEmptyKeyValue(contactInfo,
+                    EMAIL_ADDRESSES,
+                    getEmailAddresses(person.getEmailAddress()));
+            addNonEmptyKeyValue(contactInfo,
+                    PHONE_NUMBERS,
+                    getPhoneNumbers(person.getTelephoneNumber()));
+            addNonEmptyKeyValue(contactInfo, ADDRESSES, getAddresses(person.getAddress()));
+
+            personMap.put(CONTACT_INFO, contactInfo);
+            personMap.put(CUSTOM_SLOTS, getCustomSlots(person.getSlot()));
+            contactMap.put(name, personMap);
+        }
+
+        return contactMap;
+    }
+
+    private List<String> getAddresses(List<PostalAddressType> addressTypes) {
+        List<String> addresses = new ArrayList<>();
+        for (PostalAddressType address : addressTypes) {
+            ArrayList<String> addressArray = new ArrayList<>();
+
+            addNonEmptyValue(addressArray, address.getStreet());
+            addNonEmptyValue(addressArray, address.getCity());
+            addNonEmptyValue(addressArray, address.getStateOrProvince());
+            addNonEmptyValue(addressArray, address.getPostalCode());
+            addNonEmptyValue(addressArray, address.getCountry());
+
+            String addressString = StringUtils.join(addressArray, " ");
+            addresses.add(addressString);
+        }
+
+        return addresses;
+    }
+
+    private List<String> getPhoneNumbers(List<TelephoneNumberType> telephoneNumberTypes) {
+        List<String> phoneNumbers = new ArrayList<>();
+        for (TelephoneNumberType telephoneNumberType : telephoneNumberTypes) {
+            StringBuilder phoneNumber = new StringBuilder();
+
+            if (telephoneNumberType.isSetCountryCode()) {
+                phoneNumber.append(telephoneNumberType.getCountryCode() + "-");
```

Comment:
Ah yeah, this was differently formatted before but I guess I forgot to change it. Will do